lib/extract.js

```diff
@@ -11,13 +11,16 @@ function Extract (opts) {
   if (!(this instanceof Extract)) return new Extract(opts)
   tar.Parse.apply(this)
 
+  // have to dump into a directory
+  opts.type = "Directory"
```

Comment:
Definitely not necessary.  I think it's because I just accidentally put back code that was removed in the patch that introduced the regression.